docs/developers-guide/driver-changelog.md

```diff
@@ -4,6 +4,16 @@ title: Driver interface changelog
 
 # Driver Interface Changelog
 
+## Metabase 0.49.1
```

Comment:
Oh, didn't you tell me that we don't introduce driver methods in the patch releases?

Reply:
Yeah, maybe I did say that :) I can't remember how I justified it though, probably because we've never done it before. But putting it in the 0.49.0 section doesn't make much sense.